src/pages/Home/components/FavoriteRecommendation/FavoriteRecommendation.jsx

```diff
@@ -15,12 +15,9 @@ const FavoriteRecommendation = () => {
     const dislikes = getDislikes();
 
     try {
-      const res = await axios.get(
-        "http://localhost:3000/spoonacular/randomRecipe",
-        {
-          params: { number: 20 },
-        }
-      );
+      const res = await axios.get(`${env.API_URL}//spoonacular/randomRecipe`, {
```

Comment:
I suppose there should not be double slash here too?

Reply:
I've totally forgotten about this component! Thank you...!